frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/tests/premium.unit.spec.tsx

```diff
@@ -104,28 +104,26 @@ describe("Static Embed Setup phase - EE, with token", () => {
         );
       });
 
-      if (resourceType === "question") {
```

Comment:
I noticed that this check was still here, we don't need it anymore as now the flag is working on also dashboards